.vib/flux/goss/goss-wait.yaml

```diff
@@ -9,3 +10,13 @@ addr:
   tcp://podinfo:9898:
     reachable: true
     timeout: 2000
```

Comment:
Could you give a bit of context about this `podinfo`? Is this launched as part of the `Dokuwiki` chart?